src/widgets/FormControl.js

```diff
@@ -47,12 +55,17 @@ const FormControlComponent = ({
   // Save button state
   canSaveForm,
   saveButtonText,
+  // Confirm form state
+  confirmOnSave,
+  isConfirmFormOpen,
+  confirmText,
```

Comment:
do we need prop types for these

Reply:
c68c5b681fdbcb9b89f58884d0a39893695f4dee.